app/controllers/words_controller.rb

```diff
@@ -0,0 +1,57 @@
+class WordsController < ApplicationController
+
+  def index
+    @word = Word.all
+  end
+
+  def show
+    @word = Word.find(params[:id])
+  end
+
+  def new
+   @word = Word.new
+  end
+
+  def create
+    @word = Word.new(word_params)
+
+    respond_to do |format|
+      if @word.save
+        format.html { redirect_to @word, notice: 'User was successfully created.' }
```

Comment:
Please internationalize and adapt notice. This one is a bit misleading as it is not a user but word that has been created.